### PR TITLE
TWIRF-13

### PR DESCRIPTION
**Pull request by Hoverbear**
I did a thing, now I get to poke @ligthyear into reviewing it! =D

### CODE REVIEW

File: _posts/2016-09-03-this-week-in-rustfest-13-activities-and-sponsors.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "This Week in RustFest 13: A preliminary schdeule, a sponsor, and activities!"
+socialImageSrc: '/assets/posts/twirf-13-workshops.png'
+authors:
+ - hoverbear
+ - ben
+---
+
+As things get ramping up for Viewsource, and more importantly (for us!) RustFest we're happy to say we've had a busy, productive week!
```
Comment (badboy) [inline _posts/2016-09-03-this-week-in-rustfest-13-activities-and-sponsors.md]: _View Source_

File: _posts/2016-09-03-this-week-in-rustfest-13-activities-and-sponsors.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+<p style="text-align: center" markdown="1">
+[![Factor10 logo](/assets/sponsors/factor10.png)](http://www.factor10.io/)
+</p>
+
+Thank you Factor10 for your support of this community event!
+
+## Schedule In Progress
+
+This week we drafted up a schedule for all of our [super speakers](www.rustfest.eu/talks/) and we're now reaching out to them to ensure that everything sounds great to them.
```
Comment (badboy) [inline _posts/2016-09-03-this-week-in-rustfest-13-activities-and-sponsors.md]: Needs `https://` in the link (or use just `/talks/`)

File: _posts/2016-09-03-this-week-in-rustfest-13-activities-and-sponsors.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+authors:
+ - hoverbear
+ - ben
+---
+
+As things get ramping up for Viewsource, and more importantly (for us!) RustFest we're happy to say we've had a busy, productive week!
+
+## RustBridge
+
+As some of you may have noticed the [RustBridge signup post](http://discourse.opentechschool.org/t/rustbridge-workshop-sept-18th-berlin-together-with-rustfest/1849) both had a broken poll... and that was unclear about whether you needed a ticket to RustFest to join us.
```
Comment (gnunicorn) [inline _posts/2016-09-03-this-week-in-rustfest-13-activities-and-sponsors.md]: maybe an explicit sentence this is fixed now? I'd reverse the order: first it was unclear, second poll didn't work. Then the you'dont and it is fixed. Ends the paragraph nicely!

File: _posts/2016-09-03-this-week-in-rustfest-13-activities-and-sponsors.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+Well, **you don't**! You just need to sign up for the workshop! We're looking forward to seeing you there, it'll be grand.
+
+## Activities
+
+Conferences are made alive by the people who attend them, and as a community conference we want to ensure our attendees have an opportunity to plan their own activities and invite others.
+
+So this week we set up an instance of the Eurucamp Activities scheduler on Heroku! It's already under a [Heroku domain here](https://rustfest-activities.herokuapp.com/). It will live at [https://activities.rustfest.eu](https://activities.rustfest.eu) as soon as we get an SSL certificate.
```
Comment (gnunicorn) [inline _posts/2016-09-03-this-week-in-rustfest-13-activities-and-sponsors.md]: ehem. it is live on http://activities.rustfest.eu/ though ...

File: _posts/2016-09-03-this-week-in-rustfest-13-activities-and-sponsors.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+Saturday will start not earlier than 9am with a welcome, and the first keynote. As we have many talks to cover we will do them in blocks of three back-to-back with 10-15min breaks in between those blocks. Of course, there will be lunch served at the venue around mid-day within a longer break before we go on with the next blocks of talks. In the evening the [Digital Eatery](https://www.microsoft-berlin.de/de-de/berlin/the-digital-eatery/) (the café in the front) will convert to a nice hang out for us, serve dinner will and invite all participants to relax, chat and hang out (yes, it will have the XBoxes hooked up for us).
+
+Sunday starts a little later, more towards 10am, with a talk by Sonja Heinen. In "Hobby oriented programming" she will set the mode for the day, before everyone is off to their rooms and workshops. Don't worry we've planned in some travel time, so you won't be late from the talk. In the evening we invite everyone back to the venue to listen to the last talk. After a full weekend of looking into the current state and future of Rust, Marijn Haverbeke will close the conference by looking with us back to "The rust that could have been".
```
Comment (badboy) [inline _posts/2016-09-03-this-week-in-rustfest-13-activities-and-sponsors.md]: Should this be "mood of the day"?

Comment (badboy) [inline _posts/2016-09-03-this-week-in-rustfest-13-activities-and-sponsors.md]: The **R**ust that could have been

**Comment by badboy** [comment]
Two small remarks, otherwise :shipit:

**Comment by gnunicorn** [comment]
thanks @Hoverbear , @badboy .

Merging.

**Pull request merged**